torch/_dynamo/guards.py

```diff
@@ -580,14 +580,11 @@ def _print_Symbol(self, expr) -> str:
             return "0"
         if expr == 1:
             return "1"
-        expr_found = expr in (self.expr_to_tensor_ref) or (
-            expr in self.intermediary_symbols
+        assert (
+            expr in (self.expr_to_tensor_ref)
+            or (expr in self.intermediary_symbols)
+            or (expr in self.base_symbols)
```

Comment:
you know that the parens here are all unnecessary right

Reply:
I find them wonderful for readability, but yes, its all the same to python.

Reply:
`expr in (self.expr_to_tensor_ref)` lol

Reply:
fine